src/locale/pt-BR.json

```diff
@@ -18,6 +18,9 @@
       "hashtag-cancel-rent": ""
     }
   },
+  "searchbar": {
```

Comment:
[Here is the pt-BR JSON](https://github.com/antievictionmappingproject/covid-19-map/commit/e478b849949601fc3378a64845e1f44e036b2c73) with the new key for the search bar. 

p.s. I will delete this branch from the repo after this PR is merged.